lib/adoptoposs/jobs/policy.ex

```diff
@@ -14,6 +14,18 @@ defmodule Adoptoposs.Jobs.Policy do
     matches_weekday?(date, permitted_weekday)
   end
 
+  @doc """
+  Authorizes to send biweekly emails if the given date is on the `permitted_weekday`.
+  """
+  def authorize(:send_emails_biweekly, %Date{} = date, permitted_weekday) do
+    day = date.day
+
+    case matches_weekday?(date, permitted_weekday) do
+      true when day <= 7 or day in 14..20 -> true
```

Comment:
Just for style points, I'd recommend breaking the `or` statement into another case condition _or_ use an `if` clause, since using `case` on a boolean return seems a bit overkill :wink:

Reply:
@floriank , thanks for the feedback. Will make the recommended changes. 👍

Reply:
Thanks a lot for your pull request @akhanaton!
I actually quite liked the one-liner in the case statement from before :wink: 

However, I found it kind of hard to grasp with the case statement. Thinking of our future selves, maybe it will be easier to read without it? Maybe even with day ranges (1..7 and 14..20) Something like:

```elixir
(day in @first_week || day in @third_week) && matches_weekday?(date, permitted_weekday)
```
And am I mistaken or would 14..20 be the range from the last day of the 2nd week to the last but one of the 3rd week? So since we are starting the month’s days with 1, not 0, should it be 15..21?

Reply:
@paulgoetze , thanks for the feedback. I like your suggestion, it is a lot easier to figure out what's going on. You were correct on the date range (15..21).

Reply:
TIL 😉